storage-proofs/core/src/gadgets/por.rs

```diff
@@ -901,6 +901,11 @@ mod tests {
         test_private_por_input_circuit::<TestTree<PoseidonHasher, typenum::U4>>(1_172);
     }
 
+    #[test]
+    fn test_private_por_input_circuit_poseidon_oct() {
+        test_private_por_input_circuit::<TestTree<PoseidonHasher, typenum::U8>>(1_068);
+    }
+
```

Comment:
This was a missing test I added while I was editing circuit tests. It is not otherwise related to this PR.

Reply:
Yes, I also made that mistake, so v27c as a key has been burned first here: https://github.com/filecoin-project/rust-fil-proofs/pull/1132  Then here: https://github.com/filecoin-project/rust-fil-proofs/pull/1138

Go ahead and claim v27d and I'll update mine for things other than that.